src/Akkling/Spawning.fs

```diff
@@ -45,7 +45,7 @@ module System =
         let _ = Akka.Serialization.HyperionSerializer           // I don't know why, but without this system cannot instantiate serializer
         let system = ActorSystem.Create(name, config.WithFallback Configuration.extendedConfig)
         let exprSerializer = Akkling.Serialization.ExprSerializer(system :?> ExtendedActorSystem)
-        system.Serialization.AddSerializer(exprSerializer)
+        system.Serialization.AddSerializer("ExpSerializer", exprSerializer)
```

Comment:
The overload without a serializer name is deprecated, so I chose a name.

Reply:
Sure, but I'm going to remove FsPickler before the next release (it can be replaced by Hyperion for this role, which is already used). Even without FsPickler, fully fledged Akkling has 3 different serializer libs as dependencies.